code/radarr.py

```diff
@@ -1,10 +1,45 @@
 import requests
 from util import *
 
+def setupRadarr(RADARRURL, RADARRAPIKEY):
+    if LOGGING:
+        pr("Running setupRadarr function!")
+    # Define radarr dict
+    radarr = {
+        'URL': RADARRURL.rstrip("/ "),
+        'APIKEY': 'apikey=' + RADARRAPIKEY.strip(),
+    }
+    # Add API URL based on normal URL
+    radarr['APIURL'] = radarr['URL'] + '/api/v3'
+    # Test access
+    response = requests.get(radarr['URL'] + "/ping")
+    if response.status_code != 200:
+        pr("Error: Radarr response is" + str(response.status_code) + ", not 200")
+        if LOGGING:
+            # write response to file
+            dumpVar('failedRadarrResponse', response.json())
+        return False
```

Comment:
Why the 2 checks? It seems like you could just do the check below, since you're checking for a != 200 status code below anyway.

Reply:
The first check verifies that the instance is running, the second check verifies if the API key is correct. I don't know why it would be a correct URL, and not either authorized or unauthorized, but ya never know.
Though it might be worth adding better output so that the end user is aware of what in the check is failing... I'll clarify that now.